### ISSUE TITLE
a11y: Profiles are not clearly labelled in the profile switcher

### ISSUE THREAD

**Issue by pvagner** [deltachat/deltachat-desktop#5366]
- Operating System: all
- Delta Chat Version: 2.8.0
- Expected behavior: When using arrow keys to navigate in the profile switcher avatar or initial letter is displayed. Display name is most likelly shown on mouse over as a tool tip. When this is used with a screen reader only initial letter is reported
- Actual behavior: Profile display name is missing from the accessible label of each tab within the profile switcher tab list.
- Steps to reproduce the problem:
* Launch deltachat with a screen reader running
* Use tab key to navigate
* When the tab in the profile switcher page tab list is focused display name of the profile is not reported.

**Comment by WofWca** [comment]
Yes, we need to probably rewrite `<ProfileHoverInfo>` at least to make it appear on focus and not just hover, and to utilize `aria-describedby` or something.

**Comment by nicodh** [comment]
solved with https://github.com/deltachat/deltachat-desktop/pull/6170
